setup.py

```diff
@@ -57,7 +64,7 @@
         'License :: OSI Approved :: Apache Software License',
     ],
     zip_safe=False,
-    packages=find_packages(exclude=["azure"]),
+    packages=packages,
```

Comment:
Hi @jonringer , we follow [this document](https://github.com/Azure/azure-sdk-for-python/blob/master/doc/dev/packaging.md) to do azure packaging. I think we may not need to handle the python version manually, it's already been done in the `extras_require` part below.

Reply:
this only works if you also include the `python setup.py sdist` for MANIFEST.in, I have adjusted the PR